spec/requests/supers_spec.rb

```diff
@@ -1,7 +1,47 @@
 require 'rails_helper'
 
 RSpec.describe "Supers", type: :request do
-  describe "GET /index" do
-    pending "add some examples (or delete) #{__FILE__}"
+  describe "GET/ supers" do 
+    it "gets all supers" do
+    Super.create(name:"hulk", power_level:100, place_of_birth:"ohio", hobbies:"meditation", image_url:"google.com")
+    get '/supers'
+    json = JSON.parse(response.body)
+    expect(response).to have_http_status(:ok)
+    puts json
+    expect(json.length).to eq 1
```

Comment:
to improve the "fidelity" of your tests, it would be good to assert on expecte properties within your response as well